ocs_ci/deployment/deployment.py

```diff
@@ -173,6 +173,14 @@ def deploy_cluster(self, log_cli_level="DEBUG"):
             setup_netsplit(
                 tmp_path, master_zones, worker_zones, x_addr_list, arbiter_zone
             )
+        ocp_version = version.get_semantic_ocp_version_from_config()
+        if config.ENV_DATA["deploy_acm_hub_cluster"] and ocp_version >= version.VERSION_4_9:
+            try:
+                self.deploy_acm_hub()
+            except Exception as e:
+                logger.error(e)
+        else:
+            logger.warning("ACM HUB deployment is skipped")
```

Comment:
I think that we don't need to inform about this skip for non ACM deployment.

Reply:
Done